app/services/order_details/reconciler.rb

```diff
@@ -37,7 +37,7 @@ def full_errors
     # The params hash comes in with the unchecked IDs as well. Filter out to only
     # those we're going to reconcile. Returns an array of IDs.
     def to_be_reconciled
-      Hash(@params).select { |_order_detail_id, params| params[:reconciled] == "1" }
+      @params.select { |_order_detail_id, params| params[:reconciled] == "1" }
```

Comment:
Is there any case where `@params` might be nil? I think that's what the `Hash()` was mitigating.

Reply:
Handling that above in the initializer.